sdcm/utils/nemesis_utils/indexes.py

```diff
@@ -41,8 +41,10 @@ def is_cf_a_view(node: BaseNode, ks, cf) -> bool:
             return False
 
 
-def get_column_names(session, ks, cf, is_primary_key: bool = False, filter_out_collections: bool = False) -> list:
-    filter_kind = " kind in ('static', 'regular')" if not is_primary_key else "kind in ('partition_key', 'clustering')"
+def get_column_names(session, ks, cf, is_primary_key: bool = False, filter_out_collections: bool = False, filter_out_static_columns: bool = False) -> list:
+    column_types = ["\'regular\'"] if filter_out_static_columns else ["\'static\'", "\'regular\'"]
```

Comment:
I don't see the point of complicating this, why not just
`column_types = ('regular') if filter_out_static_columns else ('static', 'regular')`?
so later we don't need to join it

Reply:
I did think of just stringifying a tuple, but I don't like it because it depends on how python stringifies a tuple and it just  happens to be compatible with a CQL query, but if there are no objections in doing it that way then I'll change it to that.

Reply:
Why not use strings directly? The only thing you do with them is join
`"\'regular\'" if filter_out_static_columns else "\'static\', \'regular\'"`